mypy/stubtest.py

```diff
@@ -746,7 +814,12 @@ def verify_funcitem(
 def verify_none(
     stub: Missing, runtime: MaybeMissing[Any], object_path: List[str]
 ) -> Iterator[Error]:
-    yield Error(object_path, "is not present in stub", stub, runtime)
+    # Do not error for an object missing from the stub
```

Comment:
nit: I'd prefer if we could move this into the loop in `verify_typeinfo` and LBYL:
```
if isinstance(stub_to_verify, Missing) and is_dunder_slot_wrapper(runtime_attr):
    continue
```
That way all the logic for what gets checked within a class stays within verify_typeinfo.